component/loki/source/kubernetes_events/kubernetes_events.go

```diff
@@ -43,14 +43,16 @@ type Arguments struct {
 
 	JobName    string   `river:"job_name,attr,optional"`
 	Namespaces []string `river:"namespaces,attr,optional"`
+	LogFormat  string   `river:"log_format,attr,optional"`
```

Comment:
I think it's worth to check in `Validate` that `LogFormat` introduced is either `logfmt` or `json`